setup.py

```diff
@@ -27,8 +27,10 @@
 http://docs.python.org/2/distutils/sourcedist.html
 
 """
-import os
 from setuptools import setup, find_packages
+from sys import version_info
+import os
```

Comment:
Check https://www.python.org/dev/peps/pep-0008/#imports

See that example there:

```
import mypkg.sibling
from mypkg import sibling
from mypkg.sibling import example
```

And then we separate them in blocks:

```
1. standard library imports
2. related third party imports
3. local application/library specific imports
```

I agree that it looks weird when each block just consists of one import, then it seems like a lot of waste of space, but we just stick to it.